src/FakeItEasy/Creation/DelegateProxies/DelegateProxyGenerator.cs

```diff
@@ -147,7 +147,7 @@ bool IsAccessibleImpl(Type t)
             // Return the result if the return type is non-void
             if (!isVoid)
             {
-                bodyExpressions.Add(result);
+                bodyExpressions.Add(result!);

```

Comment:
I'm not sure it makes it more readable... Especially the `Insert`s at the end, which are a bit confusing. Anyway, I commented because I was surprised, but I wasn't asking for a change